plugins/azure-devops/src/components/EntityPageAzurePipelines/EntityPageAzurePipelines.tsx

```diff
@@ -14,18 +14,27 @@
  * limitations under the License.
  */
 
-import { useEntity } from '@backstage/plugin-catalog-react';
-import React from 'react';
-import { useRepoBuilds } from '../../hooks/useRepoBuilds';
 import { BuildTable } from '../BuildTable/BuildTable';
+import React from 'react';
+import { useAnnotationFromEntity } from '../../hooks/useAnnotationFromEntity';
+import { useBuildRuns } from '../../hooks/useBuildRuns';
+import { useEntity } from '@backstage/plugin-catalog-react';
 
 export const EntityPageAzurePipelines = ({
   defaultLimit,
 }: {
   defaultLimit?: number;
 }) => {
   const { entity } = useEntity();
-  const { items, loading, error } = useRepoBuilds(entity, defaultLimit);
+
+  const { project, repo, definition } = useAnnotationFromEntity(entity);
```

Comment:
the `use` prefix should be reserved for hooks - this is just a getter function and might be better named `getEntityAnnotations` or so.

Reply:
Got it 👍 This has been cleaned up now